.github/workflows/checks.yml

```diff
@@ -1,9 +1,7 @@
 name: Checks
 
 env:
-  STARKNET_VERSION: "0.13.0"
-  RPC_SPEC_VERSION: "0.6.0"
-  DEVNET_SHA: "1bd447d"
+  DEVNET_SHA: "79a90fd"
```

Comment:
Just in case, you can use the commit from the main branch now:
https://github.com/0xSpaceShard/starknet-devnet-rs/commit/c6ffb99597eeea58d1a6a30fc061de69e586c779